changelog.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I would prefer to keep the original pull request separated from the this feature. That would be another one.
But if we decide to include it, I would prefer to have this configuration where the maven plugin is defined (pom.xml), if this is not possible, then somewhere under resources or src/build/resources or similar.